modules/java_api/tests/ModelTests.java

```diff
@@ -1,34 +1,84 @@
-package tests;

```

Comment:
This file initially added with Windows style line endings (CRLF). Replaced with Linux style LF.